src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInModal.java

```diff
@@ -96,7 +96,7 @@ public boolean isModalDisplayed() {
 
   public void acceptOptInModal(WikiaWebDriver driver, String country, Page page) {
     setGeoCookie(driver, EU_CONTINENT, country);
-    getUrl(urlBuilder.appendQueryStringToURL(urlBuilder.getUrlForPage(page), MODAL_INSTANT_GLOBAL));
+    getUrl(page);;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)